example/model-parallel/matrix_factorization/README.md

```diff
@@ -0,0 +1,21 @@
+Model Parallel Matrix Factorization
+===================================
+
+The example walks you through a matrix factorization algorithm for recommendations and also
+demonstrates the basic usage of `group2ctxs` in `Module`, which allows one part of the model to be
+trained on cpu and the other on gpu.
+
+To run this example, first make sure you download a dataset of 10 million movie ratings available
+from [the MovieLens project](http://files.grouplens.org/datasets/movielens/) by running following command:
+
+`python get_data.py`
+
+This will download MovieLens 10M dataset under ml-10M100K folder. Now, you can run the training as follows:
+
+`python train.py --num-gpus 2`
```

Comment:
Also please add if only CPUs are available.
And, majority of users will have only 1 GPU, may be good idea to assume it.

Reply:
This example shows usage of one part of the model to be trained on CPU and other on GPU. so its necessary to have gpus on the machine. I added one line saying this:
"it is necessary to have GPUs available on the machine
to run this example."

Reply:
num_gpus is 1 to cover more general user base?